Tweak Brave description and links

TheCultLeader666 opened the pull request:
I have edited the description of Brave a little, and also removed the link to [this blog post](https://aspenuwu.me/blog/dont-use-brave/), as it only gives one side of the controversies, and it frankly seems more like an opinionated rant than an actual discussion. I have replaced it with the Controversies section of the Brave Wikipedia page, which has a much better and more professional position on it.

pluja commented:
I removed the link for the blog post and added the wikipedia controversies link. I don't want to change the "Controversial option" text as it is what it is, users may do their research and decide.

TheCultLeader666 commented:
> I removed the link for the blog post and added the wikipedia controversies link. I don't want to change the "Controversial option" text as it is what it is, users may do their research and decide.

I did leave the controversial option part, I just added on that it has some good features as a browser
